testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/transaction/SLSB1.java

```diff
@@ -88,7 +88,7 @@ public String failInFirstCall() throws Exception {
 
     /**
      * Makes two DAO calls, the transaction fails during the second DAO call.
-     * The JTA transaction is rolled back and no database changes should occur.
+     * The Jakarta Transactions rolled back and no database changes should occur.
```

Comment:
There are many examples in the document where `JTA transaction is` has been replaced with `Jakarta Transactions`. These should all read `Jakarta Transaction is`.